robot/impl/local_robot.go

```diff
@@ -709,12 +721,117 @@ func (r *localRobot) updateWeakDependents(ctx context.Context) {
 	}
 }
 
-// FrameSystemConfig returns the info of each individual part that makes up a robot's frame system.
-func (r *localRobot) FrameSystemConfig(
-	ctx context.Context,
-	additionalTransforms []*referenceframe.LinkInFrame,
-) (framesystemparts.Parts, error) {
-	return r.frameSvc.Config(ctx, additionalTransforms)
+// Config returns the info of each individual part that makes up the frame system
+// The output of this function is to be sent over GRPC to the client, so the client
+// can build its frame system. requests the remote components from the remote's frame system service.
+// NOTE(RDK-258): If remotes can trigger a local robot to reconfigure, you don't need to update remotes in every call.
```

Comment:
What does this `NOTE` mean? Can we remove it now that RSDK-258 is closed?

Reply:
Ok.  I removed these comments since the ticket has been done